deployment-wildfly.md

```diff
@@ -27,6 +26,23 @@ Or you can add it via the jboss-cli:
 /subsystem=ee:list-add(name=global-modules, value={name=se.laz.casual})
 ```
 
+## Graceful shutdown
+
+To enable graceful shutdown, wildfly has to be configured via the cli such as:
+```python
+/subsystem=ejb3:write-attribute(name=enable-graceful-txn-shutdown, value=true)
+```
+If this is not set, wildfly will not wait for in flight transactions to finish.
+That also means that `casual-jca` can't handle that as well unless wildfly is configured correctly.
+
+## Note regarding casual-jca global module not exposing its gson dependencies anymore
+
+Starting with `casual-jca` version `3.2.49`, casual-jca does not expose its `gson` dependency anymore, thus does not pollute all
+other applications with its `gson` version.
+
+That means that your own application(s) need to package whatever `gson` they want to use ( or if you have your own global module,
```

Comment:
I'm confused, how is the above example not having gson global still, as it is still part of the casual module which you then make global?